stack-packs/packs/wordpress.js

```diff
@@ -17,28 +17,28 @@ module.exports = {
     'unused-css-rules':
       'Consider reducing, or switching, the number of [WordPress plugins](https://wordpress.org/plugins/) loading unused CSS in your page. To identify plugins that are adding extraneous CSS, try running [code coverage](https://developers.google.com/web/updates/2017/04/devtools-release-notes#coverage) in Chrome DevTools. You can identify the theme/plugin responsible from the URL of the stylesheet. Look out for plugins that have many stylesheets in the list which have a lot of red in code coverage. A plugin should only enqueue a stylesheet if it is actually used on the page.',
     'uses-webp-images':
-      'Consider using a plugin/service that will automatically convert your uploaded images to the optimal formats.',
+      'Consider using a [plugin/service](https://wordpress.org/plugins/search/convert+webp/) that will automatically convert your uploaded images to the optimal formats.',
     'offscreen-images':
-      'Install a WordPress plugin that provides the ability to defer (lazy-load) any offscreen images, or switch to a theme that provides that functionality. Also consider using AMP.',
+      'Install a [WordPress plugin](https://wordpress.org/plugins/search/lazy+load/) that provides the ability to defer (lazy-load) any offscreen images, or switch to a theme that provides that functionality. Also consider using the [AMP plugin](https://wordpress.org/plugins/amp/).',
     'total-byte-weight':
       'Consider showing excerpts in your post lists (e.g. via the more tag), reducing the number of posts shown on a given page, breaking your long posts into multiple pages, or using a plugin to lazy-load comments.',
     'render-blocking-resources':
-      'There a number of WordPress plugins that can help you inline critical assets or defer less important resources. Beware that optimizations provided by these plugins may break features of your theme or plugins, so you will likely need to make code changes.',
+      'There a number of WordPress plugins that can help you [inline critical assets](https://wordpress.org/plugins/search/critical+css/) or [defer less important resources](https://wordpress.org/plugins/search/defer+css+javascript/). Beware that optimizations provided by these plugins may break features of your theme or plugins, so you will likely need to make code changes.',
     'unminified-css':
-      'A number of WordPress plugins can speed up your site by concatenating, minifying, and compressing your styles. You may also want to use a build process to do this minification up-front if possible.',
+      'A number of [WordPress plugins](https://wordpress.org/plugins/search/minify+css/) can speed up your site by concatenating, minifying, and compressing your styles. You may also want to use a build process to do this minification up-front if possible.',
```

Comment:
> You may also want to use a build process to do this minification up-front if possible

Is this possible in WP world? In my experience all CSS/JS comes from either the theme or a plugin, and you can't change those assets with a build process.